src/ConsoleWriter.php

```diff
@@ -0,0 +1,13 @@
+<?php
+namespace JakubOnderka\PhpParallelLint;
+
+class ConsoleWriter implements IWriter
```

Comment:
Why is the `IWriter` here with `I`-prefix, while `Output` above is not?

Imho the `I`-prefix comes from Microsoft products (VBScript, TypeScript) and is not fluently readable of the intent.

I like the Symfony style, where things are spelled out and thus precise and readable:
- `OutputInterface`
- `WriterInterface`

I think this project should follow that style.

Reply:
As per the original issue description, this PR includes the changes proposed in #44. This feedback actually belongs with PR #44.

Reply:
I don't see how issue titled "Switch to using GH Actions ? #44" is better than discussion where actually diff of what is discussed is actually visible.

anyway, I gave my ideas, will not my spend time arguing about the arguments nor copying the discussion.

thanks for listening!

Reply:
Hi @glensc,

These new files aren't introducing new code. There's currently a single `exceptions.php` that contains all these exceptions, which is incompatible with PSR-4. They've therefore been split in 770dc1a1b4a189132a131b1c35b13523fec2f7fc, which is a backwards-compatible change (given the user is using Composer autoloading).

The changes you're suggesting are not backwards compatible, and are therefore out-of-scope for this PR.

Again, feedback is welcome and much appreciated, but you can leave the attitude at home. 

I'll be resolving the discussions as out-of-scope.